controllers/contentlibrary/utils/utils.go

```diff
@@ -65,8 +66,24 @@ func AddContentLibraryRefToAnnotation(to client.Object, ref *imgregv1a1.NameAndK
 	if annotations == nil {
 		annotations = map[string]string{}
 	}
-	annotations[v1alpha2.VMIContentLibRefAnnotation] = string(data)
+	annotations[vmopv1.VMIContentLibRefAnnotation] = string(data)
 	to.SetAnnotations(annotations)
 
 	return nil
 }
+
+// AddImageTypeToLabel adds the image type label to the given object if the
+// label value is valid.
+func AddImageTypeToLabel(to client.Object, imageType string) error {
+	if errStrs := validation.IsValidLabelValue(imageType); errStrs != nil {
+		return fmt.Errorf("invalid image type label value %q: %v", imageType, errStrs)
+	}
+
+	labels := to.GetLabels()
+	if labels == nil {
+		labels = map[string]string{}
+	}
+	labels[vmopv1.VirtualMachineImageTypeLabel] = imageType
```

Comment:
Seems like this could added to`image.SyncStatusToLabels`, rather than adding AddImageTypeToLabel? https://github.com/vmware-tanzu/vm-operator/blob/main/pkg/util/image/status_to_label.go#L16